### PR TITLE
Removing squash4jenkins due to security issue

### PR DESCRIPTION
**Pull request by mlafaix**
Security issue is JIRA SECURITY-2525

### CODE REVIEW

File: resources/artifact-ignores.properties [hunk 1]
```diff
@@ -206,6 +206,7 @@ sms-notification                 # renamed to sms
 sonargraph-plugin = https://git.io/JJ5h1
 # deprecated (see Sonatype CLM: https://wiki.jenkins-ci.org/x/iYDPAw)
 sonatype-ci = https://wiki.jenkins.io/x/iYDPAw
+squash4jenkins                   # removal due to security issue JIRA SECURITY-2525
```
Comment (Wadeck) [inline resources/artifact-ignores.properties:209]:
```suggestion
squashtm-publisher                   # removal due to security issue JIRA SECURITY-2525
```

From https://github.com/jenkinsci/squashtm-publisher-plugin/blob/5641bf160f66be9a93bca2f7f1555c3ae3de6d08/build.gradle#L22

**Review by Wadeck — COMMENTED** [review]
We usually do not provide reference to non-yet-disclosed vulnerability to justify a plugin addition to the blocklist, but as you're also part of the maintainers of this plugin, it's fine, so 👍 from me
(I took https://henix-blog.medium.com/squash-autom-squash-devops-une-g%C3%A9n%C3%A8se-82a776208641 as a "proof" of ownership)

(not approving due to the wrong artifact id IIUC)

**Comment by daniel-beck** [comment]
@mlafaix You really should have responded in Jira first. This is pretty bad form.

**Comment by daniel-beck** [comment]
On-holding until we can properly publish an advisory.

**Comment by daniel-beck** [comment]
https://www.jenkins.io/security/advisory/2021-11-12/#SECURITY-2525

**Pull request merged**